drivers/include/periph/uart.h

```diff
@@ -163,6 +174,30 @@ void uart_poweron(uart_t uart);
  */
 void uart_poweroff(uart_t uart);
 
+/**
+ * @brief set UART mode
+ *
+ * @param[in] uart          the UART device to set parity mode
+ * @param[in] databits      number of databits
+ * @param[in] stopbits      number of stopbits
+ * @param[in] parity        parity mode
+ *
+ * @return                  0 on success
+ * @return                  -1 on invalid UART device
+ * @return                  -2 on inapplicable parity
+ */
+int uart_mode(uart_t uart, uint8_t databits,
+		uint8_t stopbits, uart_parity_t parity);
+
+/**
+ * @brief flush UART TX buffer
+ *
+ * wait until UART TX buffer is empty
```

Comment:
maybe better: 'Block until TX buffer is empty and all bytes have been shifted out.'